### PR TITLE
Update page in Cassandra admin section “cassandra_bur”

### PR DESCRIPTION
**Pull request by mvenzke-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by mvenzke-axway** [comment]
Please add Kelly as a reviewer, I'd like his input on any other warnings to add for the schema restore as mentioned in the note there.

**Comment by andreamussap** [comment]
> Please add Kelly as a reviewer, I'd like his input on any other warnings to add for the schema restore as mentioned in the note there.

Matthew, I can't add Kelly directly here but I asked him to have a look and leave his feedback

**Comment by andreamussap** [comment]
@kmercierwhite did you have a chance to review this?

File: content/en/docs/cass_admin/cassandra_bur.md [hunk 1]
```diff
@@ -137,6 +145,19 @@ do
...
+
+To restore a Cassandra schema, open `cqlsh` in the directory that contains the CQL file containing the backup, and run the following command:
+
+```
+source 'KEYSPACE_schema_backup.cql';
+```
+
+{{% alert title="Note" %}}
+This is only necessary if the schema is not already present, for example, if setting up a new cluster as a copy of an existing one.
+TODO: Add any other warnings Kelly Mercier White can think of on review.
```
Comment (alexearnshaw) [inline content/en/docs/cass_admin/cassandra_bur.md:158]: @mvenzke-axway @andreamussap I suggest removing the TODO and merging this, and @kmercierwhite can review and add more another time when he has capacity?

**Comment by mvenzke-axway** [comment]
I have a customer waiting on that, so it would be good to wait for him to merge the info.


-------- Original message --------

**Comment by andreamussap** [comment]
**Blocked** - I've spoken with @kmercierwhite He said he'll only be able to look at it once the May20 release is out.

**Pull request merged**